cli/src/interactive.rs

```diff
@@ -0,0 +1,311 @@
+use alloy::{
+    primitives::{Bytes, U256},
+    transports::http::reqwest::Url,
+};
+use anyhow::bail;
+use hdp_primitives::{
+    aggregate_fn::{integer::Operator, FunctionContext},
+    datalake::{
+        block_sampled::{
+            AccountField, BlockSampledCollection, BlockSampledCollectionType, BlockSampledDatalake,
+            HeaderField,
+        },
+        compute::Computation,
+        datalake_type::DatalakeType,
+        envelope::DatalakeEnvelope,
+        transactions::{
+            IncludedTypes, TransactionField, TransactionReceiptField, TransactionsCollection,
+            TransactionsCollectionType, TransactionsInBlockDatalake,
+        },
+        DatalakeCompute,
+    },
+    solidity_types::traits::BatchedDatalakeComputeCodecs,
+};
+use inquire::{InquireError, Select};
+use std::{path::PathBuf, str::FromStr};
+use tracing::error;
+
+use crate::common::handle_run;
+
+pub async fn run_interactive() -> anyhow::Result<()> {
```

Comment:
is this code usable? or used somwhere?

Reply:
usable, you can run by `hdp start`, magic will happens